winlogbeat/docs/configuration.asciidoc

```diff
@@ -53,16 +53,16 @@ winlogbeat:
   registry_file: C:/ProgramData/winlogbeat/.winlogbeat.yaml
 --------------------------------------------------------------------------------
 
-NOTE: The forward slashes in the path are automatically changed to back slashes
-for Windows compatibility. Either forward or back slashes may be used. Forward
+NOTE: The forward slashes (/) in the path are automatically changed to backslashes
+(\) for Windows compatibility. You can use either forward or backslashes. Forward
 slashes are easier to work with in YAML because there is no need to escape them.
 
 ===== event_logs
 
-In YAML terminology, `event_logs` is defined as a list of dictionaries. Each
-entry in the list defines an event log to monitor as well as any information to
```

Comment:
@dedemorton Following up on your question about what I meant here w.r.t. "as well as any information to be associated with the event log (filter, tags,...".  I was just being vague because all of the additional configuration options are net yet defined. We will have to revisit this section as the code matures.